### PR TITLE
Add chromium container

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

**Comment by dagguh** [comment]
4e0376e is a fixup and should be squashed

File: src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/Ryuk.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.testcontainers.dockerclient.DockerClientProviderStrategy
+import java.util.*
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ *  Test containers 1.9.1 doesn't work with Bitbucket pipelines.
+ *  Test containers start a docker with Ryuk. Ryuk needs to mount the docker socket, and this is forbidden in Bitbucket pipelines.
+ *  Ryuk isn't a critical feature. It makes after test docker container cleanup more robust.
+ *  This class is a hack to disable the feature.
+ *  Sorry for the hack, but I neither understand how to fix Ryuk problems, neither want to fork and release forked test-containers.
```
Comment (bsideup) [inline src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/Ryuk.kt]: Soon, we will make it possible to disable Ryuk if you env allows it :) See https://github.com/testcontainers/testcontainers-java/issues/1023

Good that you have a workaround meanwhile!

Comment (bsideup) [reply src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/Ryuk.kt]: Well, it used to work but then there was some change on BB side and it got broken.
We track the current state of BitBucket here:
https://github.com/testcontainers/testcontainers-java/issues/700#issuecomment-447581970

Comment (bsideup) [reply src/main/kotlin/com/atlassian/performance/tools/dockerinfrastructure/Ryuk.kt]: @wyrzyk FYI https://github.com/testcontainers/testcontainers-java/issues/700#issuecomment-457541087

**Pull request merged**